scRNA-seq/03-normalizing_scRNA.Rmd

```diff
@@ -38,109 +47,78 @@ Now let's set up the files we will be using:
-To do this, we are going to use some single-cell RNA-seq R packages called `scater` and `scran`. 
-
-![](diagrams/full-length_2.png)
-
-The functions in these packages require the data to be in their own special object type (not an uncommon thing for R packages to do) called `SingleCellExperiment`.
-So first, we are going to set up our data in this format.
-To learn more about `SingleCellExperiment` objects and how it works with Bioconductor packages, we recommend this article: [Amezquita *et al.* 2020.](https://www.nature.com/articles/s41592-019-0654-x).
-
-
-```{r}
-# Set up the Single Cell Experiment object 
-gbm_sce <- SingleCellExperiment(list(counts = as.matrix(sc_data_filtered[, -1])))
-
-# Store the gene names in this object
-rownames(gbm_sce) <-  sc_data_filtered$gene
-```
-
-In the intro-to-R-tidyverse module notebook, `01-intro-to-base_R.Rmd`, we discuss base R object types, but there are some 'special' object types that are package-specific. 
-`SingleCellExperiment` objects, for example, are what a lot of single-cell analysis R packages use, so we will try to get acquainted with them. 
-
-Use this chunk we've set up below to do some exploring of the `SingleCellExperiment` object you just made.
-
-```{r Explore_SCEstructure, live = TRUE, eval = FALSE}
-# The SingleCellExperiment is a special type of object used by scater and scran
-# R packages, explore its properties here. 
-# Also check it out in the environment tab.
-gbm_sce
+bladder_sce <- readr::read_rds(filtered_sce_file)
+sc_metadata <- readr::read_csv(metadata_file, guess_max = 10000)
 ```
 
-Below is a figure from [Amezquita *et al.* (_bioRxiv._ 2019)](https://www.biorxiv.org/content/10.1101/590562v1) (the preprint version of the above-linked paper) that shows the general structure of `SingleCellExperiment` objects. 
-
-![](figures/sce_structure.png)
-
-Note that each function which transforms the `SingleCellExperiment` object can store its results back into the object (or a new copy of the object, but that tends to be a waste of space).
-Each of these transformation functions also have a corresponding extraction function that allow you extract (or set) those results from the `SingleCellExperiment` object. 
-
-For more information on `SingleCellExperiment` objects, check out this [excellent vignette](https://bioconductor.org/packages/devel/bioc/vignettes/SingleCellExperiment/inst/doc/intro.html)
-and/or [Amezquita *et al.* (2020)](https://www.nature.com/articles/s41592-019-0654-x)
-
 ### Adding more metadata to the SCE object
 
+In a previous notebook, we used some of the data from the Tabula Muris project analysis to tag our cells for visualization.
+Note that we would normally **NOT** have this information until later in the analysis pipeline!
+Nonetheless, adding it here will be useful for visualizing the results of our normalization (and demonstrating how one might add metadata to the SingleCellExperiment object).
+
```

Comment:
```suggestion
Because this is a well-studied data set, we actually have some cell type information for this data set.

Note that we would normally **NOT** have this information until later in the analysis pipeline!
Nonetheless, adding it here will be useful for visualizing the results of our normalization (and demonstrating how one might add metadata to the SingleCellExperiment object).

```

Since we plan on taking out the cell type gene expression plots in the second notebook on filtering, we should probably change this description here.

Reply:
> I like the changes that you made here and thinks all looks good in terms of content. I added a few areas where I thought we could use some clarity, but other than that I didn't think there was much need for any major changes.
> 
> The one thing I did want to ask was about the lack of live typing in this notebook? It looks like there's only a handful of chunks that are live typing. I think we could probably increase that and use the smaller chunks as live to keep people engaged.

I agree there should probably be more live typing here; I don't think I was too focused on that when making the current edits. I think I planned to come back through with those changes a bit later, once I had all the content sorted & running properly.

I like your other edits too. I'm going to take a pass at wording tomorrow (and getting all the suggestions where they go, since github only lets you comment on lines near changes).